packages/core/src/lib/hooks/useParent.ts

```diff
@@ -1,9 +1,22 @@
-import { getContext } from 'svelte'
-import type { Writable } from 'svelte/store'
+import { createObjectStore } from '../../lib'
+import { getContext, setContext } from 'svelte'
+import { type Writable } from 'svelte/store'
 import type { Object3D } from 'three'
 
 export type ThrelteParentContext = Writable<Object3D | undefined>
 
+const key = Symbol('threlte-hierarchical-parent-context')
+
 export const useParent = () => {
-  return getContext<ThrelteParentContext>('threlte-hierarchical-parent-context')
+  return getContext<ThrelteParentContext>(key)
+}
+
+export const setParent = (context: ThrelteParentContext) => {
+  return setContext(key, context)
+}
+
+export const createParentContext = (ref: Object3D | undefined) => {
+  const context = createObjectStore(ref)
```

Comment:
This is new and worth calling out: before the T component was just passing a writable to this context, but  `createObjectStore`, but `HierarchicalObject` was passing `createObjectStore(ref)`.

I fixed this inconsistency since it seems like what we would want, but I'm curious if there's anything I'm potentially missing.

Reply:
Afaik, the objectStore is just a store that doesn't emit updates when the uuid of a newly set object doesn't change, so all good 👍